src/nisarqa/validate/xml_spec_checker/policy.py

```diff
@@ -207,6 +207,91 @@ def ignored_xml_annotation_attributes() -> set[str]:
     return {"lang", "app"}
 
 
+def common_nisar_projection_attributes() -> set[str]:
+    """
+    Attributes common to NISAR "projection" Datasets but not required by CF-1.7.
+
+    These Attributes have been added to all "projection" Datasets by
+    NISAR convention. However, they are not required by CF-1.7 conventions
+    for either UTM or polar-stereographic projections.
+
+    Returns
+    -------
+    set[str]
```

Comment:
Typically we would document types like this more informally in the docstring (since NumPy docstring conventions predate this style of type annotation)

```suggestion
    set of str
```